lib/orbit/badge_serial.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Orbit.BadgeSerial do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Orbit.Employee
+
+  @type t :: %__MODULE__{
+          badge_serial: String.t()
```

Comment:
Should this `@type` contain an in-struct reference to `Employee`, since it's in the schema?

Reply:
I was sort of torn on this one. The association goes both ways, but I think in practice the right way to use these is going to be to reference the `Employee` primarily and look at the `Employee`'s `badge_serials` value, rather than the other way around. Anyhow, all of that said, I don't think it hurts anything to add it as optional, see 7af289d.